src/tower/__main__.py

```diff
@@ -47,7 +47,13 @@ def on_draw(self):
 
         self.scene.draw()
 
+    def get_asset_path(self, file_name: str) -> str:
+        """Returns the asset path as a string."""
 
+        assets_dir = Path(__file__).resolve().parent.parent / "assets"
```

Comment:
I think `resolve` typically should be the last method used to avoid weird file paths

Reply:
Why did we add the call to `resolve` here? It wasn't there originally.

Reply:
Assumed we need it because we want the absolute path.

> Make the path absolute, resolving any symlinks. A new path object is returned: